packages/host/app/components/matrix/room.gts

```diff
@@ -289,7 +291,10 @@ export default class Room extends Component<Signature> {
       },
     });
 
-    if (!this.autoAttachedFileUrl) {
+    if (
+      !this.autoAttachedFileUrl ||
+      this.operatorModeStateService.state.submode === Submodes.Interact
+    ) {
```

Comment:
Let's put this conditional logic inside the `autoAttachedFileUrl` getter instead. i.e. have it return undefined unless this.operatorModeStateService.state.submode is code mode